src/pages/Insights/InsightsPage.module.scss

```diff
@@ -25,15 +25,16 @@
 }
 
 .mainSection {
-  padding: 0 16px;
+  padding: 0;
```

Comment:
@haritonasty  What was the reason for removing `16px`?

Reply:
because on mobile FeaturedInsights block has full screen width